{"name": "test_login_03", "status": "passed", "description": "错误的密码为yoyy123456", "start": 1584783241473, "stop": 1584783241817, "uuid": "2abfa683-a9f4-468e-9973-b4dc45a4fef1", "historyId": "54a8bf10d2a14801f9bb563dbfe38c02", "testCaseId": "c4dcd1e1235303eabc98c65b38314763", "fullName": "case.mul_form_data.test_login.TestLogin#test_login_03", "labels": [{"name": "feature", "value": "登录的接口"}, {"name": "story", "value": "传入正确的用户名,错误的密码"}, {"name": "parentSuite", "value": "case.mul_form_data"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "S28RZMS68F89XOJ"}, {"name": "thread", "value": "6748-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.mul_form_data.test_login"}]}